packages/elements/stencil.config.ts

```diff
@@ -31,16 +31,9 @@ export const config: Config = {
     },
     angularOutputTarget({
       componentCorePackage: '@inovex.de/elements',
-      directivesProxyFile:
-        '../elements-angular/elements/src/directives/proxies.ts',
-      directivesUtilsFile:
-        '../elements-angular/elements/src/directives/proxies-utils.ts',
-      directivesArrayFile:
-        '../elements-angular/elements/src/directives/proxies-list.ts',
-    }),
-    angularOutputTargetFix({
-      directivesUtilsFile:
-        '../elements-angular/elements/src/directives/angular-component-lib/utils.ts',
+      directivesProxyFile: `${angularDiretivesPath}/proxies.ts`,
+      directivesUtilsFile: angularDiretivesPath,
```

Comment:
It seems that we use the same utils [file](https://github.com/ionic-team/stencil-ds-output-targets/blob/master/packages/angular-output-target/angular-component-lib/utils.ts) like the npm package. 
I suggest to clean up our code base and use the default.